CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 # Changelog
 
-### 117.0.3 [#1884]([https://github.com/openfisca/openfisca-france/pull/1884)
+### 117.0.4 [#1891]([https://github.com/openfisca/openfisca-france/pull/1891)
+
+* Changement mineur.
+* Zones impactées :
+  - parameters/prelevements_sociaux/hsup_exo
+* Détails :
+ - Retire un paramètre non utilisé
```

Comment:
```suggestion
 - Retire un paramètre qui est un extrait de formule, non référencé et non utilisé
```

Reply:
Nous pourrions aussi indiquer qu'il s'agit de l'harmonisation avec [baremes-ipp : prelevements_sociaux/pss.yaml](https://framagit.org/french-tax-and-benefit-tables/baremes-ipp-yaml/-/blob/0c26f8df373e0bc52fa6494e2dcaab313f83dc8e/parameters/prelevements_sociaux/pss.yaml)